tests/unit/test_local_test_utils.py

```diff
@@ -0,0 +1,199 @@
+# Copyright Contributors to the Packit project.
+# SPDX-License-Identifier: MIT
+
+from pathlib import Path
+from unittest.mock import patch
+
+from packit.utils.local_test_utils import LocalTestUtils
+
+
+class TestBuildTmtCmd:
+    """Test cases for _build_tmt_cmd function"""
+
+    def test_basic_command_structure(self):
```

Comment:
Afaict, packit doesn't use class-based organization of tests. Use fixtures and plain function-based tests unless instructed otherwise by the core packit team.